src/app.js

```diff
@@ -48,29 +49,18 @@ app.get('/filmer/:movieId', async (req, res) => {
   }
 });
 
-
 app.get('/api/screenings', async (req, res) => {
   const upcomingScreenings = await getUpcomingScreenings(cmsAdapter);
   res.send(upcomingScreenings);
 });
 
 app.get('/api/movies/:movieID/screenings', async (req, res) => {
-  try {
-    const movieId = req.params.movieID;
-    const screenings = await loadScreenings(movieId);
-    const currentTime = new Date().getTime();
-    const upcomingScreenings = screenings.data.filter((screening) => {
-      return new Date(screening.attributes.start_time).getTime() >= currentTime;
-    });
-
-    res.json({ data: upcomingScreenings });
-  } catch (error) {
-    if (error.message === 'Screening not found') {
-      res.status(404).render('screening404');
-    } else {
-      res.status(500).send('Internal Server Error');
-    }
-  }
+  const movieId = req.params.movieID;
+  const upcomingMovieScreenings = await getUpcomingMovieScreenings(
+    cmsAdapter,
+    movieId
+  );
+  res.json(upcomingMovieScreenings);
```

Comment:
Like the incorporation of the cmsAdapter 😀
But how do you handle error?

Reply:
i was hoping to just outsource it ;) 
but it is added now :)
thanks for noticing